sormas-ui/src/main/java/de/symeda/sormas/ui/exposure/ExposuresField.java

```diff
@@ -94,13 +95,21 @@ protected void updateColumns() {
 			table.setVisibleColumns(
 				EDIT_COLUMN_ID,
 				COLUMN_EXPOSURE_TYPE,
+				COLUMN_EXPOSURE_ROLE,
```

Comment:
Exposure role doesn't need a generated column because all you're doing here is to make a toString. You can remove `COLUMN_EXPOSURE_ROLE` (and the code to add the generated column) and just replace it with `ExposureDto.EXPOSURE_ROLE`